packages/app-desktop/ElectronAppWrapper.ts

```
@@ -596,10 +605,10 @@ export default class ElectronAppWrapper {
 		syncPending: boolean,
 	) {
 		if (syncPending) {
-			dispatch({ type: 'QUIT_SYNC_DIALOG_OPEN' });
-		} else {
-			this.quit();
-		}
+	        dispatch({ type: 'QUIT_SYNC_DIALOG_OPEN' });
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)